docs/migration-guide/to-17.md

```diff
@@ -82,6 +83,30 @@ Using the CLI:
 +stylelint "**/*.css" --custom-formatter @csstools/stylelint-formatter-github
 ```
 
+### Removed `resolveNestedSelectors` option from `selector-class-pattern`
+
+We've removed the `resolveNestedSelectors` option from `selector-class-pattern` because it's incompatible with [standard CSS Nesting](https://drafts.csswg.org/css-nesting/), which doesn't concatenate strings.
+
+For example, the following is invalid CSS:
+
+<!-- prettier-ignore -->
+```scss
```

Comment:
[suggestion] Minor. If we'd like to show that it's *invalid* CSS, specifying it as CSS, not SCSS, might be better:

````suggestion
```css
````